packages/node/src/node.ts

```diff
@@ -8,6 +8,9 @@ import { Base } from "@logtail/core";
 
 import { getStackContext } from "./context";
 
+const CONTEXT_OBJECT_MAX_DEPTH_DEFAULT : number = 20;
+const CONTEXT_OBJECT_MAX_DEPTH_WARN : boolean = process.env.LOGTAIL_CONTEXT_OBJECT_MAX_DEPTH_WARN === 'true';
```

Comment:
I'm not sure about using ENV variables for this even though it should be fine since this is just for node.
I think I might prefer these to be sent in the `ILogtailOptions` object in the constructor. Any thoughts @gyfis ?

Reply:
Yep, agree with `ILogtailOptions`

Reply:
Hi, thanks for your feedbacks, I will do that 👍